constellations/gemini-user/src/afh/afh.sass

```diff
@@ -0,0 +1,64 @@
+@import "../utils/constants.sass"
+
+@media only screen and (max-width: 700px)
+    #view-afh
+        width: 100%
+
+        & h1, .description
+            text-align: center
+
+        & li
+            width: 90%
+            margin: $spacing1x auto
+
+        .tab button
+            font-size: 8px
+
```

Comment:
the font size of the buttons isn't changing

Reply:
line 52 should be `className` instead of `class`